spec/javascript/packages/document-capture/components/document-capture-warning-spec.jsx

```diff
@@ -47,7 +47,12 @@ describe('DocumentCaptureWarning', () => {
     }
   }
 
-  function renderCcontent(isFailedDocType, isFailedResult, inPersonUrl) {
+  function renderContent({
+    isFailedDocType,
+    isFailedResult,
+    isFailedSelfieLivenessOrQuality,
+    inPersonUrl,
+  }) {
```

Comment:
should we add types to this? or add default values? looks like they're not all required properties
```suggestion
    isFailedDocType,
    isFailedResult,
    isFailedSelfieLivenessOrQuality,
    inPersonUrl,
  } : {
    isFailedDocType: boolean;
    isFailedResult: boolean;
    isFailedSelfieLivenessOrQuality?: boolean;
    inPersonUrl: string;
  }) {
```

Reply:
Thanks! Added the default, but I'm thinking that changing the file to typescript might be too much for now.